api/src/main/java/com/xwiki/googleapps/internal/GoogleAppsManagerImpl.java

```diff
@@ -0,0 +1,1439 @@
+import org.xwiki.component.manager.ComponentManager;
+import org.xwiki.component.phase.Disposable;
+import org.xwiki.component.phase.Initializable;
+import org.xwiki.component.phase.InitializationException;
+import org.xwiki.environment.Environment;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.model.reference.DocumentReferenceResolver;
+import org.xwiki.model.reference.ObjectReference;
+import org.xwiki.query.Query;
+import org.xwiki.query.QueryException;
+import org.xwiki.query.QueryManager;
+import org.xwiki.stability.Unstable;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeRequestUrl;
+import com.google.api.client.googleapis.auth.oauth2.GoogleTokenResponse;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.InputStreamContent;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.store.FileDataStoreFactory;
+import com.google.api.services.drive.Drive;
+import com.google.api.services.drive.DriveScopes;
+import com.google.api.services.drive.model.File;
+import com.google.api.services.drive.model.FileList;
+import com.google.api.services.people.v1.PeopleService;
+import com.google.api.services.people.v1.PeopleServiceScopes;
+import com.google.api.services.people.v1.model.EmailAddress;
+import com.google.api.services.people.v1.model.Person;
+import com.google.gdata.client.docs.DocsService;
+import com.google.gdata.data.MediaContent;
+import com.google.gdata.data.media.MediaSource;
+import com.xpn.xwiki.XWiki;
+import com.xpn.xwiki.XWikiContext;
+import com.xpn.xwiki.XWikiException;
+import com.xpn.xwiki.doc.XWikiAttachment;
+import com.xpn.xwiki.doc.XWikiDocument;
+import com.xpn.xwiki.objects.BaseObject;
+import com.xpn.xwiki.web.XWikiRequest;
+import com.xpn.xwiki.web.XWikiResponse;
+
+/**
+ * Set of methods accessible to the scripts using the GoogleApps functions.
+ *
+ * @version $Id$
+ * @since 3.0
+ */
+@Component
+@Singleton
+public class GoogleAppsManagerImpl
+        implements GoogleAppsManager, Initializable, Disposable
+{
+
+    //  -----------------------------  Lifecycle ---------------------------
+
+    @Inject
+    private Provider<XWikiContext> xwikiContextProvider;
+
+    @Inject
+    private QueryManager queryManager;
+
+    @Inject
+    private Environment environment;
+
+    @Inject
+    @Named("current")
+    private DocumentReferenceResolver<String> documentResolver;
+
+    @Inject
+    @Named("user")
+    private DocumentReferenceResolver<String> userResolver;
+
+    @Inject
+    private Logger log;
+
+    @Inject
+    private ComponentManager componentManager;
+
+    @Override
+    public void initialize() throws InitializationException
```

Comment:
Class fields should come before the class methods..

Reply:
Should now be cleaned out by the automatic code arrangement. I indeed had the wrong setting wired. Thanks. This now means that the separation between the fields is not present anymore.